### ISSUE TITLE
SEO tags rendering inside body

### ISSUE THREAD

**Issue by mcclaskiem** [nystudio107/craft-seomatic#800]
### Describe the bug
After seeing our google search results being malformed, I did some investigating and noticed that all of the tags output by SEOmatic are being placed inside of the opening `<body>` tag.

**Additional Info:**
This site was converted from Craft 2.7 and the equivalent SEOmatic plugin and is using a migrated database. I do have the `{% hook 'seomaticRender' %}` in my `_layout.twig` file but also saw that auto rendering is enabled by default so not sure if this is even necessary. I did try removing this hook and and also disabling auto-rendering with the hook placed and no difference was noticed in the behavior. 

### To reproduce
Steps to reproduce the behaviour:
1. Navigated to any page
2. Open Inspector
3. Notice SEOmatic markup is placed within the opening of the `<body>` tag.

### Expected behaviour
SEOmatic tags would be rendered in the `<head>` tag on all pages for normal users and search engine crawlers.

### Screenshots
<img width="586" alt="Screen Shot 2020-12-19 at 9 38 09 PM" src="https://user-images.githubusercontent.com/6970287/102703874-bc569e00-4242-11eb-9a17-8d72557036ed.png">

### Versions
- Plugin version: `3.3.26`
- Craft version: `3.5.17.1`

**Comment by khalwat** [comment]
The only time I've ever seen this happen has been when the HTML is malformed. SEOmatic uses Yii2's standard routines for adding tags.

Please ensure your HTML validates by running it through the [WC3 Validator](https://validator.w3.org/), and you don't have multiple `<head>` tags or multiple `<body>` tags or lacks closing tags for either of those.

Also ensure you don't have any JavaScript that could be interfering or manipulating the DOM in some way that is causing this.

Happy to help you diagnose what is wrong here, but it's very likely to be what I've mentioned above.

The `{% hook 'seomaticRender' %}` is no longer necessary with SEOmatic for Craft 3.

**Comment by mcclaskiem** [comment]
@khalwat Thanks for getting back to me and offering to help. After validating, I got a bunch of errors with additional title tags being in the `<head>`. I then realized that I had a symbdol defs file in the head with a few SVGs inside it, each with it's own `<title>` tag. After moving that include to inside the body, the SEO matic mark up is now rendering properly in the head! This was not directly related to multiple head or body tags but Im assuming the title tags were enough to throw off Yii2's tag adding.

Appreciate the tip and help here!
